packages/server/src/main/java/software/uncharted/terarium/hmiserver/repository/data/ModelConfigRepository.java

```diff
@@ -9,5 +9,8 @@
 
 @Repository
 public interface ModelConfigRepository extends PSCrudSoftDeleteRepository<ModelConfiguration, UUID> {
-	List<ModelConfiguration> findByModelIdAndDeletedOnIsNullAndTemporaryFalse(UUID modelId, Pageable pageable);
+	List<ModelConfiguration> findByModelIdAndDeletedOnIsNullAndTemporaryFalseOrderByCreatedOnAsc(
```

Comment:
theseNamesGetPrettyCrazy

Reply:
They are _SQL_ like method for hibernates to do the right request.